src/main/java/core/basesyntax/service/impl/CsvWriter.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.StorageDao;
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class CsvWriter implements WriterService {
+    private static final String TITLE_LINE = "fruit,quantity";

```

Comment:
same here, maybe we don't need title, and if we need, we should be able to pass it through constructor